models/configuration.go

```diff
@@ -40,17 +17,11 @@ type Configuration struct {
 	// The password for the PCF API.
 	PCFPassword string `json:"pcf_password"`
 
-	// verifyOpts is intentionally lower-cased so it won't be stored in JSON.
-	// Instead, this struct is expected to be created from NewConfiguration
-	// so that it'll populate this field.
-	verifyOpts *x509.VerifyOptions
-}
+	// The maximum minutes old a login request's signing time can be.
+	// This is configurable because in some test environments we found as much as 2 hours of clock drift.
+	LoginMaxMinOld int `json:"login_max_minutes_old"`
 
-// VerifyOpts returns the options that can be used for verifying client certificates,
-// including the CA certificate pool.
-func (c *Configuration) VerifyOpts() (x509.VerifyOptions, error) {
-	if c.verifyOpts == nil {
-		return x509.VerifyOptions{}, errors.New("verify options are unset")
-	}
-	return *c.verifyOpts, nil
+	// The maximum minutes ahead a login request's signing time can be.
+	// This is configurable because in some test environments we found as much as 2 hours of clock drift.
+	LoginMaxMinAhead int `json:"login_max_minutes_ahead"`
```

Comment:
I didn't find any strangeness where login attempts were from times far in the future, but just in case, I made this configurable too.